src/SettingsCart.tsx

```diff
@@ -38,6 +38,9 @@ export  const SettingsCart: React.FC<SettingsCartParams> = (props) => {
     if (!values.name) {
       errors.name = t('cart-name-is-required');
     }
+    if(values.name.length > 250) {
```

Comment:
For our own understanding, is there still a check on the back-end to ensure cart name's cannot exceed 250 characters? Will the API throw an error if someone hit it directly?

Reply:
@shaunmaharaj this is the API response when greater than 255
So I am not sure why we do the value `250` here?

`"errors": [
        {
            "status": 400,
            "source": "data.name",
            "title": "string_lte",
            "detail": "String length must be less than or equal to 255",
            "meta": {
                "value": "ijoifjaoisdjoifadjfaidjfioamdiofjoaidmscfjadiosmficajdsiomcfoaijsdoifadijifcajdiojfocajdiosmfiojdismfjeorijgoijroeijimgeorifgoejrifmeiorjfmoigjreiogmiremgjreimgjiremgieroimgoirjeoigjreiojgmoermgirejgmoremgiorejgiogriajgoermjgoiejgirigjrijgirjgjggrrrrrrrrra"
            }
        }
    ]`

Reply:
@kaiyuancheng My feeling here was a client-side check is done to save on requests (similar as we check client-side for emails first).  
@yasiloghmani Where was this decision made for the client-side check? Did it come from UX as a requirement?

Reply:
@shaunmaharaj This is the ticket https://elasticpath.atlassian.net/browse/MT-5253. I thought that it is client side check.